backend/src/main/java/sysc4806group25/monkeypoll/controller/AccountController.java

```diff
@@ -70,6 +70,16 @@ public ResponseEntity<String> registerAccount(@RequestBody RegisterRequest regis
         throw new ResponseStatusException(HttpStatus.CONFLICT, "An account with that email already exists.");
     }
 
```

Comment:
Let's add an endpoint that gets the current user, it will help fix some of the issues you were having

Reply:
Thanks for the endpoint, I agree that this is the way to go